docs/src/components/MarkerCamundaExtension.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const MarkerCamundaExtension = () => {
+  return (
+      <p>
+        <span style={{
+          backgroundColor: '#25c2a0',
```

Comment:
🔧 You could align this with the brand color codes, e.g. Greenmunda: `#26D07C`

Reply:
If we discuss colors / align this I'd rather use a neutral or warning color than a "awesome, go for this" color, cf. [here](https://miro.com/app/board/uXjVPOXAyQ4=/?moveToWidget=3458764536100903050&cot=14). We don't want to extra "encourage" use of vendor extensions. Some may be safe to adopt across our tooling (i.e. adding a custom function), others will make you run into trouble (i.e. the not FEEL spec compliant backtick string).

Reply:
I'll change the color to `#FC5D0D` (Orangemunda).